src/helpers/overview-table/value-threshold.ts

```diff
@@ -0,0 +1,49 @@
+import { add } from 'date-fns';
+import getDayKey from '../get-day-key';
+import { OverviewData } from './data-provider';
+import { PrimaryOverviewDataType } from './data-types';
+
+export interface OverviewValueThreshold {
+    label: string;
+    minimumValue?: number;
+    maximumValue?: number;
+}
+
+export type OverviewThresholdDaysLookup = { [key: string]: string[] };
+
+export const NotEnteredThreshold = 'Not Entered';
+
+export function computeThresholdDays(startDate: Date, endDate: Date, overviewData: OverviewData<PrimaryOverviewDataType>): OverviewThresholdDaysLookup {
+    const lookup: OverviewThresholdDaysLookup = {};
```

Comment:
Can unit tests be added for this?

Reply:
Yes.  Will do.  I'll also work on adding them to a number of other bits of code as well.

Reply:
I have added a bunch of unit tests for the helper functions.